applications/CompressiblePotentialFlowApplication/tests/naca0012_small_transonic_test/naca0012_small.mdpa

```diff
@@ -0,0 +1,210 @@
+Begin ModelPartData
```

Comment:
Why are you adding twice this mdpa? Please remove one of them. You can create an auxiliary_files folder in the tests folder to store it if it is going to be used by several examples.